compressed_depth_image_transport/src/compressed_depth_publisher.cpp

```diff
@@ -45,7 +45,7 @@
 #include <vector>
 #include <sstream>
 
-constexpr int kDefaultPngLevel = 9;
+constexpr int kDefaultPngLevel = 3;
```

Comment:
is there any reason to change the default behaviour ? I would like to avoid change this because it might affect other users.

The other change LGTM

Reply:
The default was changed in future releases so I just matched them, and when I tested it a while back I think I remember 9 compression giving me problems (I think taking too long that it was reducing the frame rate). I would need to run it again to check, but since it's a param we can just change it back